cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -401,6 +403,11 @@ private void copyStream(ReadOnlyDataSource from, DataSource to, String fromName,
             "Source for IIDM identifiers",
             SOURCE_FOR_IIDM_ID_MRID,
             List.of(SOURCE_FOR_IIDM_ID_MRID, SOURCE_FOR_IIDM_ID_RDFID));
+    private static final Parameter UNESCAPE_IDENTIFIERS_PARAMETER = new Parameter(
```

Comment:
Maybe is easier to read if we change to affirmative? (`USE_ESCAPED_IDENTIFIERS`?)

Reply:
or `HANDLE_ESCAPED_IDENTIFIERS`

Reply:
Is `ESCAPE_IDENTIFIERS` okay for you?

Reply:
Is `DECODE_ESCAPED_IDENTIFIERS` okay for you? `HANDLE` is a bit vague for me

Reply:
ok for me